CONTRIBUTING.md

```diff
@@ -0,0 +1,11 @@
+# How to Contribute
+
+The ansible-documention project is [GPL-3.0 licensed](COPYING) and accepts contributions through
+GitHub pull requests.
+
+## Certificate of Origin
+
+By contributing to this project you agree to the Developer Certificate of
```

Comment:
```suggestion
By contributing to ansible-documentation, you agree to the Developer Certificate of
```

Reply:
Do you see benefit for other projects to reference this? If so, we should keep it generic?

Reply:
Other projects can copy this into their own repository and change the project name.